tinytag/tinytag.py

```diff
@@ -835,17 +835,17 @@ def _determine_duration(self, fh):
                 is_info = fh.read(4)  # check INFO header
                 if is_info != b'INFO':  # jump over non-INFO sections
                     fh.seek(subchunksize - 4, os.SEEK_CUR)
-                    continue
```

Comment:
Essentially this continue goes to beginning of the loop instead of dropping down to the desired chunk_header read.  By changing to if / else we trigger the reading of the next chunk_header on each loop.